### ISSUE TITLE
check_every does not default to 1m

### ISSUE THREAD

**Issue by alkar** [concourse/concourse#2280]
# Bug Report

Unless `check_every` is specified, resources are only checked once when the pipeline is unpaused.

Example:
```
resources:
- name: 10s
  type: time
  source:
    interval: 10s
jobs:
- name: check-every-10s
  plan:
  - get: 10s
    trigger: true
```

To fix this simply specify `check_every: 5s` for the resource.
However, updating the pipeline does not seem to work - deleting the pipeline and re-creating it will do the trick.

I expected the resource to be checked every `1m` as per the default value mentioned in the documentation for resources.

* Concourse version: 3.14.0
* Deployment type (BOSH/Docker/binary): N/A
* Infrastructure/IaaS: Kubernetes
* Browser (if applicable): N/A
* Did this used to work? Docs seem to imply it did.

First seen: https://github.com/concourse/time-resource/issues/28

**Comment by alkar** [comment]
Closing - this is a configuration issue.

**Comment by bhutkovskyysos** [comment]
@alkar How is this a configuration issue? This seems to be a bug within concourse I think. I got the same problem with the latest Concourse 5.3.0. Only after I deleted and re-created the pipeline the `check_every` setting took affect.

**Comment by propalparolnapervom** [comment]
Same problem, Concourse 5.3.0
New `check_every` value applies only after pipeline re-creation.

**Comment by benchristel** [comment]
@vito I think this is a real issue. My team ran into it today on Concourse 5.3.0. Any chance this could be re-opened/investigated? We don't want to delete and recreate our pipelines since we'd lose build history.

cc @roicos

**Comment by vito** [comment]
Please open a new issue with steps to reproduce. The OP closed their own issue, I don't think it should be reopened.

**Comment by benchristel** [comment]
This was a configuration issue on my end as well. We were setting `check_every` in the `resource_type` config instead of for each resource.
